bcda/service/resources_test.go

```diff
@@ -0,0 +1,142 @@
+package service
+
+import (
+	"github.com/CMSgov/bcda-app/bcda/constants"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type ResourcesTestSuite struct {
+	suite.Suite
+}
+
+// Run all suite tests
+func TestResourcesTestSuite(t *testing.T) {
+	suite.Run(t, new(ResourcesTestSuite))
+}
+
+func (s *ResourcesTestSuite) TestSupportsDataType() {
+	tests := []struct {
+		name         string
+		dataType     DataType
+		dataTypeName string
+		expected     bool
+	}{
+		{
+			"Valid Adjudicated Type",
+			DataType{Adjudicated: true, PreAdjudicated: true},
+			constants.Adjudicated,
+			true,
+		},
+		{
+			"Valid Pre-Adjudicated Type",
+			DataType{Adjudicated: true, PreAdjudicated: true},
+			constants.PreAdjudicated,
+			true,
+		},
```

Comment:
Since both permissions are true in these will the test really ensure the appropriate one was being checked?  I would suggest making the one not being tested false.